Governance (#77)
hyperspy/hyperspy-website: community/governance.rst

core developers team, Steering Council (SC) and a Project’s Benevolent dictator for life
(BDFL) [#bdfl]_. We view this governance model as the formalization of what we are
already doing, rather than a change in direction.

Roles and Responsibilities
--------------------------

The Community
~~~~~~~~~~~~~

The HyperSpy community consists of anyone using or working with the project
in any way.

Contributors
~~~~~~~~~~~~

A community member can become a contributor by interacting directly with the
project in concrete ways, such as:

- proposing a change to the code via a
  `GitHub pull request <https://github.com/hyperspy/hyperspy/pulls>`_;
- reporting issues on our
  `GitHub issues page <https://github.com/hyperspy/hyperspy/issues>`_;
- proposing a change to the documentation, or
  `demos <https://github.com/hyperspy/hyperspy-demos>`_ via a
  GitHub pull request;
- discussing the design of HyperSpy in existing
  `issues <https://github.com/hyperspy/hyperspy/issues>`_ and
  `pull requests <https://github.com/hyperspy/hyperspy/pulls>`_;
- reviewing `open pull requests <https://github.com/hyperspy/hyperspy/pulls>`_

> 

Reviewer: ```suggestion
- teaching new users the usage of HyperSpy in tutorial sessions, e.g. in the framework of international conferences
```

comprehensive view of their contributions. This will include but is not limited
to code, code review, infrastructure work, chat participation, community
help/building, education and outreach, contributions to other projects of the HyperSpy
ecosystem, design work, etc. We are deliberately
not setting arbitrary quantitative metrics (like "100 commits in this repo") to
avoid encouraging behavior that plays to the metrics rather than the project's
overall well-being. We want to encourage a diverse array of backgrounds, viewpoints
and talents in our team, which is why we explicitly do not define code as the
sole metric on which council membership will be evaluated.

If a Council member becomes inactive in the project for a period of one year,
they will be considered for removal from the Council. Before removal, the inactive
Member will be approached by the chair to see if they plan
on returning to active participation. If not, they will be removed immediately
upon a Council vote. If they plan on returning to active participation soon,
they will be given a grace period of one year. If they don't return to active

> participation within that time period they will be removed by vote of the Council

Reviewer: ```suggestion
participation within that time period, they will be removed by vote of the Council
```

without further grace period. All former Council members can be considered for
membership again at any time in the future, like any other Project Contributor.
Retired Council members will be listed on the project website, acknowledging the
period during which they were active in the Council.

The Council reserves the right to eject current Members, other than the BDFL,
if they are deemed to be actively harmful to the project's well-being, and
attempts at communication and conflict resolution have failed.

> The HyperSpy steering council may be contacted at ``EMAIL ADDRESS`` or via the
> `@hyperspy/steering-council <https://github.com/orgs/hyperspy/teams/steering-council>`_
> GitHub team.

Author: ```suggestion
The HyperSpy steering council may be contacted at ``EMAIL ADDRESS``.
```
Considering the scope of the steering council, advertising a GitHub team is not necessary, however, it will be good to have an email address. For example, scikit-image uses https://groups.io:
https://scikit-image.org/docs/stable/skips/1-governance.html#steering-council

Conflict of interest
~~~~~~~~~~~~~~~~~~~~

It is expected that the SC Members will be employed at a wide range of companies,

> universities and non-profit organizations. Because of this, it is possible that

Reviewer: I would list the academic employers first :-)

```suggestion
It is expected that the SC Members will be employed at a wide range of universities, 
research institutes, companies and non-profit organizations. Because of this, it is possible that
```

consensus seeking to come to a resolution. In the unlikely event that there is
still a deadlock, the proposal will move forward if it has the support of a
simple majority of the SC.

Decisions (in addition to adding core developers and SC membership as above)
are made according to the following rules:

- **Minor documentation changes**, such as typo fixes, or addition / correction of a
  sentence, require approval by a core developer *and* no disagreement or requested
  changes by a core developer on the issue or pull request page (lazy
  consensus). Core developers are expected to give "reasonable time" to others
  to give their opinion on the pull request if they're not confident others
  would agree.

- **Code changes and major documentation changes** require agreement by *one*
  core developer *and* no disagreement or requested changes by a core developer
  on the issue or pull-request page (lazy consensus). For all changes of this type,
  core developers are expected to give "reasonable time" after approval and before
  merging for others to weigh in on the pull request in its final state.

- **Changes to the public API** require a dedicated issue on our

>   `issue tracker <https://github.com/hyperspy/hyperspy/issues>`_ and follow the

Author: ```suggestion

- **Suggestion for changes to this governance model or our mission and principles** are welcome and require a dedicated issue on our `issue tracker <https://github.com/hyperspy/hyperspy-website/issues>`_. Changes to this governance requires approval of the Steering Council.
```
